core/templates/pages/preferences-page/form-fields/subject-interests.component.ts

```diff
@@ -65,13 +65,11 @@ export class SubjectInterestsComponent {
   }
 
   validInput(value: string): boolean {
-    // The following regex matches all the special characters and numbers.
-    let invalidRegex = new RegExp(
-      '[~`!@#$%^&*()_+={[}\\]|\\:;"\'<,>.?/\\-0-9]');
+    // The following regex matches only lowercase
+    // alphabetic characters and spaces.
+    let validRegex = new RegExp('^[a-z\\s]*$');
```

Comment:
Do we / should we have similar validation in the backend?

Reply:
there was a similar validation in the backend which I have removed in this [PR](https://github.com/oppia/oppia/pull/19633/files) as suggested by you.

![image](https://github.com/oppia/oppia/assets/119884665/19a174e7-0978-425c-afeb-1e1ad9bc59de)

![image](https://github.com/oppia/oppia/assets/119884665/566e72e9-3db7-45f3-8b47-8331ce6ab73e) in [constants.ts](https://github.com/oppia/oppia/blob/develop/assets/constants.ts)

Reply:
OK, thanks, that's seems fine then.